### DISCUSSION TITLE
Run wfreerdp-server.exe Failed!

### DISCUSSION THREAD

**Discussion by GGDK1** [FreeRDP/FreeRDP#10499] [General]
After starting wfreerdp-server.exe (version 2.11.7) on Windows 11 and attempting to connect from Ubuntu using xfreerdp, the following log appears in wfreerdp-server: "Could not find dfmirage mirror driver! Is it installed?".
 After installing dfmirage-setup-2.0.301.exe, the log changes to "Could not attach display device!". 
How should I configure it to run wfreerdp-server.exe properly?

**Comment by akallabeth** [comment]
well, the  `CMake` and startup warnings should have given you a clue and a simple [issue search](https://github.com/FreeRDP/FreeRDP/issues?q=is%3Aopen+is%3Aissue+label%3Awindows+label%3Aserver) the answer ;)

the windows server component is not maintained and nobody has interest in improving it.

(main reason is most likely that you can just use windows `RDP` server instead)

**Reply by GGDK1** [reply]
thx，error  fixed

**Comment by GGDK1** [comment]
#if _WIN32_WINNT >= 0x0602
#define WITH_DXGI_1_2 1
#endif

Use  _WIN32_WINNT  = 0x0602， then wfreerdp-server will  work with  dgxi
